presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -710,13 +710,18 @@ public void testExecute()
     {
         try (Connection connection = createConnection()) {
             try (Statement statement = connection.createStatement()) {
-                assertTrue(statement.execute("SELECT 123 x, 'foo' y"));
+                assertTrue(statement.execute("SELECT 123 x, 'foo' y, NULL z"));
                 ResultSet rs = statement.getResultSet();
                 assertTrue(rs.next());
                 assertEquals(rs.getLong(1), 123);
                 assertEquals(rs.getLong("x"), 123);
+                assertEquals(rs.wasNull(), false);
                 assertEquals(rs.getString(2), "foo");
                 assertEquals(rs.getString("y"), "foo");
+                assertEquals(rs.wasNull(), false);
+                assertEquals(rs.getLong(3), 0);
+                assertEquals(rs.getObject("z"), null);
+                assertEquals(rs.wasNull(), true);
                 assertFalse(rs.next());
```

Comment:
We should add a couple more checks: that calling `wasNull()` works for both the ordinal and named getters, and changing the order so it checks `x`, `z`, `y` (so false, true, false).